substrate/frame/staking-async/src/pallet/mod.rs

```diff
@@ -419,13 +414,6 @@ pub mod pallet {
 	#[pallet::storage]
 	pub type ValidatorCount<T> = StorageValue<_, u32, ValueQuery>;
 
-	/// Any validators that may never be slashed or forcibly kicked. It's a Vec since they're
-	/// easy to initialize and the performance hit is minimal (we expect no more than four
-	/// invulnerables) and restricted to testnets.
-	#[pallet::storage]
-	pub type Invulnerables<T: Config> =
```

Comment:
Good to mention in the prdoc that this storage and the call `set_invulnerables` is removed.